src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@use 'sass:list';
+@import 'variables';
```

Comment:
The `@import` rule is considered deprecated in favor of `@use` and `@forward`. It's recommended to use `@use` instead of `@import` for better scoping and to avoid potential conflicts.